tests/src/tools/ReadyToRun.SuperIlc/Program.cs

```diff
@@ -0,0 +1,32 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.CommandLine.Builder;
+using System.CommandLine.Invocation;
+using System.Threading.Tasks;
+
+namespace ReadyToRun.SuperIlc
+{
+    class Program
+    {
+        static async Task<int> Main(string[] args)
+        {
+            var parser = CommandLineOptions.Build()
+                
+                .UseHelp()
+                .UseParseDirective()
+                .UseDebugDirective()
+                .UseSuggestDirective()
+                .RegisterWithDotnetSuggest()
```

Comment:
> RegisterWithDotnetSuggest

This will keep trying to launch a process to do the registration. It does not look like something we need/want.

Reply:
That's a good point - the middleware has some neat features I was trying out. We can probably drop the debug directive too.